README.rst

```diff
@@ -57,23 +57,68 @@ our homepage_.
 .. |weblate| image:: https://weblate.bubu1.eu/widgets/searxng/-/searxng/svg-badge.svg
    :target: https://weblate.bubu1.eu/projects/searxng/
 
+Contact
+"""""""
+
+Matrix: `#searxng:matrix.org <https://matrix.to/#/#searxng:matrix.org>`_
+
+IRC:  `#searxng on libera.chat <https://web.libera.chat/?channel=#searxng>`_ which is bridged to Matrix.
 
-SearXNG is a fork of `searx`_:
 
-* `the differences with searx <https://github.com/searxng/searxng/issues/46>`_
-* `exhaustive changelog <https://github.com/searxng/searxng/wiki/Changes-from-version-1.0.0>`_
+Differences with searx
+""""""""""""""""""""""
+
+SearXNG is a fork of `searx`_.
+Here some of the changes:
 
 .. _searx: https://github.com/searx/searx
 
 
-Contact
-"""""""
+User experience
+~~~~~~~~~~~~~~~
 
-Matrix: `#searxng:matrix.org <https://matrix.to/#/#searxng:matrix.org>`_
+-  Huge update of the simple theme:
 
-IRC:  `#searxng on libera.chat <https://web.libera.chat/?channel=#searxng>`_ which is bridged to Matrix.
+   -  usable on desktop, tablet and mobile.
+   -  light and dark versions (you can choose in the preferences)
+   -  on the way to support RTL languages.
+
+-  the translations are up to date, you can contribute `here`_
+-  the preferences page has been updated:
+
+   -  you can see which engines is reliable or not. 
+   -  engines are grouped inside each tab.
+   -  each engine has a description
+
+-  it is easier to report a bug of an engine...
+-  ... but you can also disable the recording of the metrics on the
+   server.
+
+Setup
+~~~~~
+
+-  the docker image is built for the amd64, arm64 and arm/v7
+   architectures
+-  you don't need Morty to proxy the images.
+-  on the way to embedded filtron.
+-  up to date installation scripts
+
+Contributions are easier
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  readable debug log
+-  contributions to the themes are made `easier`_
+-  a lot of code cleanup and bug fixes
+-  the Python, NPM and Docker dependencies are up to date.
+
+Plugins
+~~~~~~~
+
+-  the hostname_replace.py plugin can remove results
```

Comment:
The filename should prolly be monospace ... also "`_replace.py` can remove results" sounds like we don't take naming very seriously. If it can do more than replacing it should be called sth different IMO.

Reply:
what about `hostname_filter`? Or we can remove this line from README.rst until the issue is fixed.

Reply:
Rewriting and removing are two separate things. Since the plugin can do both, I think it makes sense to just call it `hostname.py`.

Reply:
I have removed this section, so the naming issue can be solved after this PR.

Reply:
Alright I opened #768 for the naming issue.